scripts/generate-conda-recipe.py

```diff
@@ -0,0 +1,86 @@
+import yaml
+import sys
+
+class RecipeDumper(yaml.SafeDumper):
```

Comment:
@floralatin ask me for the python review

I am not sure to understand what this dumper do and why it is needed

Reply:
In order to add a line break between top-level objects and ignore aliases, so i write it

Reply:
Okay I am not sure that it is conda best practice ...

Reply:
Do other project do like this ?

Reply:
> Do other project do like this ?
In jina core [create-conda-recipe.py](https://github.com/jina-ai/jina/tree/master/scripts#:~:text=create%2Dconda%2Drecipe.py)